### ISSUE TITLE
Forbidden: permission denied

### ISSUE THREAD

**Issue by NicolasPelletier** [Autodesk/aomi#162]
I am trying to push to vault and get the following error:
```
Connected to https://<InternalServer>:8200 as <MyUsername>, v0.7.0
Unexpected error: hvac.exceptions.Forbidden
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/aomi-1.10.3-py2.7.egg/aomi/cli.py", line 467, in main
    action_runner(parser, args)
  File "/usr/local/lib/python2.7/site-packages/aomi-1.10.3-py2.7.egg/aomi/cli.py", line 434, in action_runner
    aomi.seed_action.seed(client.connect(args), args)
  File "/usr/local/lib/python2.7/site-packages/aomi-1.10.3-py2.7.egg/aomi/seed_action.py", line 43, in seed
    .fetch(vault_client) \
  File "/usr/local/lib/python2.7/site-packages/aomi-1.10.3-py2.7.egg/aomi/model/context.py", line 356, in fetch
    existing = getattr(vault_client, b_class.list_fun)()
  File "build/bdist.linux-x86_64/egg/hvac/v1/__init__.py", line 341, in list_audit_backends
    return self._get('/v1/sys/audit').json()
  File "build/bdist.linux-x86_64/egg/hvac/v1/__init__.py", line 944, in _get
    return self.__request('get', url, **kwargs)
  File "build/bdist.linux-x86_64/egg/hvac/v1/__init__.py", line 986, in __request
    self.__raise_error(response.status_code, text, errors=errors)
  File "build/bdist.linux-x86_64/egg/hvac/v1/__init__.py", line 996, in __raise_error
    raise exceptions.Forbidden(message, errors=errors)
Forbidden: permission denied
```

**Comment by otakup0pe** [comment]
This is a side effect of recent changes to handle tunable mount points... Will take a look and see how we can handle this more gracefully.

**Comment by otakup0pe** [comment]
Pretty sure this has been fixed (in #164), let me know if that is not the case.
